src/ui/browserAction/popupPage.js

```diff
@@ -172,6 +172,27 @@ export class BrowserActionPopup extends LitElement {
       this.currentSiteContext = new_cntxt;
     };
 
+    const getExclusionStringElem = (origin) => {
+      const originPlaceholder = "dummyString";
+      const localizedString = tr("exludePageFor", originPlaceholder);
+      // Create a new <p> element
+      const el = document.createElement("p");
+
+      // Replace "dummyString" with <span>origin</span>
+      const parts = localizedString.split(originPlaceholder);
```

Comment:
This will create a new node on every render()
Using a lit template result will simply cause lit to update the text of the span :) 

``` 
    const parts = localizedString.split(originPlaceholder);
      return html`
          <p>
          ${parts.at(0)}
          <span class="bold">${origin}</span>
          ${parts.at(-1)}
         </p>
        `    
```

Reply:
done!